src/color/creating_reading.js

```diff
@@ -242,6 +237,18 @@ p5.prototype.brightness = function(c) {
  * </code>
  * </div>
  */
+
+/**
+ * @method color
+ * @param  {Number|String} v1      red or hue value relative to
+ *                                 the current color range, or a color string
+ * @param  {Number}        v2      green or saturation value
+ *                                 relative to the current color range
+ * @param  {Number}        v3      blue or brightness value
+ *                                 relative to the current color range
+ * @param  {Number}        [alpha]
```

Comment:
I've added some assertions to the preprocessor that actually *enforce* that the same params have the same type, and that subsequent declarations of the same param have an empty description, among other things. This may be a bit draconian, but for now I wanted to make sure that the expectations of the overload code are well-understood.

I think it can also help prevent confusion--for instance, if a contributor were to provide a different description for the second version of `alpha` and the docs build system *didn't* complain, the contributor might then get confused when they don't see their description anywhere in the docs!